### PR TITLE
Merge bash scripts to main branch

### PR DESCRIPTION
**Pull request by wolender**
I added all of the tasks from "Scripting Bash" module.

### CODE REVIEW

File: caesar/caesar.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#use example
```
Comment (vasichkin) [inline caesar/caesar.sh]: Please, add task definition

**Comment by vasichkin** [comment]
Please, organise code in folders - task1, task2, etc.

**Comment by wolender** [comment]
I fixed project structure

File: bash/task1/fibonacci.sh
Comment (vasichkin) [inline bash/task1/fibonacci.sh:1]: Approved

File: bash/task2/oppars.sh
Comment (vasichkin) [inline bash/task2/oppars.sh:1]: C11233:task2 vkhomenko$ ./oppars.sh -o * -n 1 -1 -d
User: vkhomenko
Script: ./oppars.sh
Operation: oppars.sh
Numbers: 2
The ressult is : 1

File: bash/task3/fizzbuzz.sh
Comment (vasichkin) [inline bash/task3/fizzbuzz.sh:1]: Approved

File: bash/task4/caesar.sh
Comment (vasichkin) [inline bash/task4/caesar.sh:1]: C11233:task4 vkhomenko$ cat input.txt
hello world
C11233:task4 vkhomenko$ ./caesar.sh -s 10 -i input.txt -o cipher.txt
C11233:task4 vkhomenko$ cat cipher.txt
rovvy gybvn
C11233:task4 vkhomenko$ ./caesar.sh -s -10 -i cipher.txt -o restored.txt
C11233:task4 vkhomenko$ cat restored.txt
hello ]oXld

File: bash/task5/task5.sh
Comment (vasichkin) [inline bash/task5/task5.sh:1]: C11233:task5 vkhomenko$ ./task5.sh -i input.txt -o result.txt -s XX dd
No outputfile
C11233:task5 vkhomenko$ ./task5.sh -i input.txt -o output.txt -r
./task5.sh: line 37: tac: command not found

File: bash/task6/task6.sh
Comment (vasichkin) [inline bash/task6/task6.sh:1]: Approved

**Review by vasichkin — CHANGES_REQUESTED** [review]
Some tasks failed

**Comment by wolender** [comment]
fixed all the issues

File: bash/task2/oppars.sh
Comment (vasichkin) [inline bash/task2/oppars.sh:1]: Approved

File: bash/task4/caesar.sh
Comment (vasichkin) [inline bash/task4/caesar.sh:1]: Approved

File: bash/task5/task5.sh
Comment (vasichkin) [inline bash/task5/task5.sh:1]: C11233:task5 vkhomenko$ cat input.txt
Hello student
HOW are you?
XX to replace by YY
!@#$^&
the END
C11233:task5 vkhomenko$ ./task5.sh -i input.txt -o result.txt -l
C11233:task5 vkhomenko$ cat result.txt

C11233:task5 vkhomenko$

**Review by vasichkin — CHANGES_REQUESTED** [review]

File: bash/task5/task5.sh
Comment (vasichkin) [inline bash/task5/task5.sh:1]: Approved

**Review by vasichkin — APPROVED** [review]

**Pull request merged**